packages/apputils-extension/style/contextualhelp.css

```diff
@@ -2,22 +2,15 @@
  * Copyright (c) Jupyter Development Team.
  * Distributed under the terms of the Modified BSD License.
  */
-:root {
-  --jp-private-shortcuts-table-padding-vertical: 9px;
-  --jp-private-shortcuts-table-padding-right: 14px;
-  --jp-private-shortcuts-key-padding-horizontal: 5px;
-  --jp-private-shortcuts-key-padding-vertical: 3px;
-}
-
-tr,
-td {
-  padding-bottom: var(--jp-private-shortcuts-table-padding-vertical);
-  padding-top: var(--jp-private-shortcuts-table-padding-vertical);
-  padding-right: var(--jp-private-shortcuts-table-padding-right);
-}
 
 .jp-ContextualShortcut-TableRow {
+  --jp-private-shortcuts-row-padding-vertical: 9px;
+  --jp-private-shortcuts-row-padding-right: 14px;
```

Comment:
Should [relative length units](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Values_and_units#lengths) be used here so that for example when font size is changed the padding would also change?